base/common/src/main/java/com/netscape/certsrv/key/KeyResource.java

```diff
@@ -30,7 +30,8 @@ public Response listKeys(@QueryParam("clientKeyID") String clientKeyID,
                                  @QueryParam("maxTime") Integer maxTime,
                                  @QueryParam("start") Integer start,
                                  @QueryParam("size") Integer size,
-                                 @QueryParam("realm") String realm);
+                                 @QueryParam("realm") String realm,
+                                 @QueryParam("ownerName") String ownerName);
```

Comment:
Similarly, the param name probably should be called `owner` as well.

Reply:
fixed.